tutorial/source/bayesian_regression.ipynb

```diff
@@ -408,6 +430,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
+    "Bayesian nonlinear regression can also be implemented analogously by lifting neural network modules."
```

Comment:
you mention nns here for the first time, even though theyre used above.  is this to distinguish 'actual' nns as approximators for strictly non-linear functions?  (ie the purpose of this PR)